presto-product-tests/src/main/java/io/prestosql/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -146,4 +147,17 @@ public void testInsertIntoPartitionedWithSerdeProperty()
         assertThat(query("INSERT INTO " + tableNameInDatabase + " SELECT 1, 'presto', '2018-01-01'")).containsExactly(row(1));
         assertThat(query("SELECT * FROM " + tableNameInDatabase)).containsExactly(row(1, "presto", "2018-01-01"));
     }
+
+    @Test
+    public void testInsertIntoTransactionalTable()
+    {
+        onHive().executeQuery("" +
+                "CREATE TABLE test_insert_into_transactional_table(a bigint)" +
+                "CLUSTERED BY(a) INTO 4 BUCKETS STORED AS ORC TBLPROPERTIES ('transactional'='true', 'bucketing_version'='1')");
```

Comment:
Why use bucketing here? That seems to complicate the test for no reason.

Reply:
trannsactional tables need to be bucketed before Hive 3